src/simbridge-client/src/components/McduServerClient.ts

```diff
@@ -8,9 +8,10 @@ import { ClientState } from './ClientState';
  * Class to communicate with the SimBridge MCDU server
  */
 export class McduServerClient {
+    public static ip = ():string => NXDataStore.get('CONFIG_SIMBRIDGE_IP', 'localhost');
     public static port = ():string => NXDataStore.get('CONFIG_SIMBRIDGE_PORT', '8380');
 
-    public static url = ():string => `ws://127.0.0.1:${this.port()}/interfaces/v1/mcdu`.replace(/\s+/g, '');
+    public static url = ():string => `ws://${this.port()}:${this.port()}/interfaces/v1/mcdu`.replace(/\s+/g, '');
```

Comment:
I don't think that address is going to work...

Reply:
Oops, thanks